core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -576,7 +576,7 @@ var ExplorationEditorMainTab = function() {
       'Add/Update Solution modal takes too long to close');
   };
 
-  // INTERACTIONS
+  // INTERACTIONS.
```

Comment:
I'm not sure the pattern enforced by the linter makes sense here. This is a heading, not a sentence, so it doesn't make sense to require punctuation. We could require that all comments be complete sentences, but that seems overly prescriptive, as this comment looks perfectly clear and well-formatted to me. I actually think writing it as a complete sentence, e.g. "Below we define functions that work with interactions", would be less readable.

Reply:
@U8NWXD Then I think it is better to ignore comments like this. What do you think?

Reply:
@Hudda do you mean by adding `@ts-ignore` to these comments? If so then I worry that we'll have to add `@ts-ignore` all over the place, which defeats the purpose of the linter. I think the core issue here is that the rule itself:

> The comment should have space after //, must start with a capital letter and must have a valid punctuation at the end of the comment.

is too narrow. What if we decided on a format for header comments and then enforced that? Here are some ideas:

1. Add hyphens before and after the comment like this: `// ----- INTERACTIONS -----`
2. All headers are in all-caps like `// INTERACTIONS`
3. Headers are followed by a line of hyphens:
   ```
   // INTERACTIONS
   // ------------
   ```
4. There are many more ...

Reply:
@U8NWXD No, I am thinking of excluding comments with only two words in them
In that way we also exclude comments like these `// Paragraph Separator`

Reply:
@Hudda Why two words? I think we can have header-like comments with more than two words. For example:

```
// ----- Options to be passed to cucumber -----
```

from [here](https://github.com/oppia/oppia/pull/9574/files?file-filters%5B%5D=.js&file-filters%5B%5D=.py&file-filters%5B%5D=.ts&file-filters%5B%5D=dotfile&owned-by%5B%5D=U8NWXD#diff-be9307c112ba8cb518cbee7579fe85b4L351)?

Reply:
> Add hyphens before and after the comment like this: // ----- INTERACTIONS -----

@U8NWXD Then, this option seems best. What do you think?

Reply:
@Hudda sounds good to me! Let me know when you've made the change and I'll take another pass

Reply:
@U8NWXD I've already made the changes PTAL!